docs/tutorials/google/colab.ipynb

```diff
@@ -187,49 +187,13 @@
     }
    ],
    "source": [
+    "import cirq_google as cg\n",
+    "\n",
     "# The Google Cloud Project id to use.\n",
     "project_id = '' #@param {type:\"string\"}\n",
+    "processor_id = \"\" #@param {type:\"string\"}\n",
     "\n",
-    "if project_id == '':\n",
-    "    import os \n",
-    "    if 'GOOGLE_CLOUD_PROJECT' not in os.environ:\n",
-    "        raise Exception(\"Please setup project_id in this cell or set the `GOOGLE_CLOUD_PROJECT` env var to your project id.\")\n",
-    "    project_id = os.environ['GOOGLE_CLOUD_PROJECT']\n",
-    "else: \n",
-    "    os.environ['GOOGLE_CLOUD_PROJECT'] = project_id\n",
-    "\n",
-    "def authenticate_user():\n",
-    "    \"\"\"Runs the user through the Colab OAuth process.\n",
-    "    \n",
-    "    Checks for Google Application Default Credentials and runs interactive login \n",
-    "    if the notebook is executed in Colab. In case the notebook is executed in Jupyter notebook\n",
-    "    or other IPython runtimes, no interactive login is provided, it is assumed that the \n",
-    "    `GOOGLE_APPLICATION_CREDENTIALS` env var is set or `gcloud auth application-default login`\n",
-    "    was executed already.\n",
-    "    \n",
-    "    For more information on using Application Default Credentials see \n",
-    "    https://cloud.google.com/docs/authentication/production\n",
-    "    \"\"\"\n",
-    "    in_colab = False\n",
-    "    try:\n",
-    "        from IPython import get_ipython\n",
-    "        in_colab = 'google.colab' in str(get_ipython())\n",
-    "    except: \n",
-    "        # Notebook is not executed within IPython. Assuming external authentication.\n",
-    "        return \n",
-    "\n",
-    "    if in_colab: \n",
-    "        from google.colab import auth      \n",
-    "        print(\"Getting OAuth2 credentials.\")\n",
-    "        print(\"Press enter after entering the verification code.\")\n",
-    "        auth.authenticate_user(clear_output=False)\n",
-    "        print(\"Authentication complete.\")\n",
-    "    else: \n",
-    "        print(\"Notebook is not executed with Colab, assuming Application Default Credentials are setup.\") \n",
-    "\n",
-    "authenticate_user()\n",
-    "\n",
-    "print(\"Successful authentication to Google Cloud.\")"
+    "(device, line_length), sampler, _ = cg.engine.engine_sampler.get_device_sampler(project_id, processor_id)"
```

Comment:
line_length unused

Reply:
Will be fixed by in the next commit as I switch over to the dataclass. Resolving this and the following one for now.